kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -301,7 +301,11 @@ private synchronized void manageConfiguredInterfaces(List<String> configuredInte
                 continue;
             }
 
-            manageConfiguredInterface(iface, properties);
+            try {
+                manageConfiguredInterface(iface, properties);
+            } catch (Exception e) {
```

Comment:
Can we have a more specific catch? `DBusException` and `DBusRuntimeException`.

```suggestion
            } catch (Exception e) {
```